adapter/internal/api/apis_impl.go

```diff
@@ -337,12 +354,22 @@ func ApplyAPIProjectInStandaloneMode(payload []byte, override *bool) error {
 
 	// TODO: (renuka) optimize to update cache only once when all internal memory maps are updated
 	for vhost, environments := range vhostToEnvsMap {
-		updateAPI(vhost, apiInfo, apiProject, environments)
+		_, err := updateAPI(vhost, apiInfo, apiProject, environments)
+		if err != nil {
+			return err
+		}
 	}
 	return nil
```

Comment:
can we use just return here since we have named return parameters ?

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2482